R/tostring.R

```diff
@@ -165,20 +209,22 @@ setMethod("toString", "MatrixPrintForm", function(x,
     c(.do_inset(div, inset), "", txt)
 }
 
-.footer_inset_helper <- function(x, div, inset) {
+.footer_inset_helper <- function(footers_v, div, inset) {
     div_done = FALSE
-    rfn <- x$ref_footnotes
-    footer_txt <- .do_inset(x$ref_footnotes, inset)
+    fter <- footers_v$main_footer
+    prvf <- footers_v$prov_footer
+    rfn <- footers_v$ref_footnotes
+    footer_txt <- .do_inset(rfn, inset)
     if(any(nzchar(footer_txt)))
         footer_txt <- .inset_div(footer_txt, div, inset)
-    if(any(nzchar(all_footers(x)))) {
-        if(any(nzchar(prov_footer(x))))
-            provtxt <- c(if(any(nzchar(main_footer(x)))) "",
-                         prov_footer(x))
+    if(any(nzchar(footers_v))) {
```

Comment:
This is a list, so please don't call nzchar on it directly, the result can be counter-intiutive:
```
> nzchar(list(""))
[1] FALSE
> nzchar(list("", c("", "")))
[1] FALSE  TRUE
```

Other than that, having the inset happen on the wrapped things makes sense to me. Be sure you're wrapping to the correct widths though, titles should wrap as before, but main footer and referential footnotes have `inset` less space and need to be wrapped accordingly